packages/react/src/hooks/useChain.ts

```diff
@@ -7,6 +7,19 @@ import { QUERY_KEYS } from '../utils';
 
 import { useProvider } from './useProvider';
 
+/**
+ * `useChain` is a React hook that returns the chain info for the current provider.
+ *
+ * @returns {Promise<ChainInfo | null>} ChainInfo for the current provider.
```

Comment:
no hook will return directly the value. it will always follow the same pattern:
`useChain` returns `{ chain }`
`useBalance` returns `{ balance }` 

And so on